pages/common/mpg321.md

```diff
@@ -0,0 +1,29 @@
+# mpg321
+
+> High Performance MPEG 1.0/2.0/2.5 Audio Player for Layer 1, 2, and 3.
+> Mpg321 was written (sometime in 1999) to be a drop-in replacement for the (previously) non-free mpg123 player.
+> More information: <http://mpg321.sourceforge.net/>.
+
+- Play an audio source exactly N times (N=0 means forever):
+
+`mpg321 -l {{N}} {{path/to/file_a|URL}} {{path/to/file_b|URL}} {{...}}`
+
+- Play a whole directory (recursively):
```

Comment:
Is there a need to keep "recursively" in parentheses ? The sentence works fine without it.

Reply:
removed the parentheses.